datafusion/core/tests/data/aggregate_test_100_with_order.csv

```diff
@@ -0,0 +1,101 @@
+c1,c2,c3,c4,c5,c6,c7,c8,c9,c10,c11,c12,c13
```

Comment:
This file is same with aggregate_test_100.csv file. However, rows are reordered according to c1 ASC